### ISSUE TITLE
Option to clear boss bars on world change or death.

### ISSUE THREAD

**Issue by mibby** [ryandw11/UltraBar#15]
@ryandw11 Would it be possible to add an option to clear boss bars when a player dies or switches worlds?

I have a special item that runs this command when they use it.

```
      - 'console: bar send %player% Extraction_Inbound... Blue 0 300'
```

After a certain amount of time (5 minutes), the player is teleported back to spawn. However if they die, the boss bar still exists and displays the timer when they respawn.

**Comment by ryandw11** [comment]
Possibly. I am trying to think how that would work though. I will look into adding that as a feature though. Also the syntax of the bar command was updated in the 2.0 release.

**Comment by mibby** [comment]
I noticed. :) I'll be updating to v2.0 today with my 1.14 update.

If a way to clear boss bars on death or world change could be looked into and implemented, that would be wonderful!

**Comment by ryandw11** [comment]
Found a way. Will be released in the next version (2.1).
